### PR TITLE
Commerce 4407 item selector 4 account

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Review by dsitu — APPROVED** [review]

File: modules/apps/commerce/commerce-account-item-selector-web/src/main/resources/META-INF/resources/account_user_item_selector.jsp [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+<%
+CommerceAccountUserItemSelectorViewDisplayContext commerceAccountUserItemSelectorViewDisplayContext = (CommerceAccountUserItemSelectorViewDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+SearchContainer<User> commerceAccountUserSearchContainer = commerceAccountUserItemSelectorViewDisplayContext.getSearchContainer();
+String itemSelectedEventName = commerceAccountUserItemSelectorViewDisplayContext.getItemSelectedEventName();
+PortletURL portletURL = commerceAccountUserItemSelectorViewDisplayContext.getPortletURL();
+%>
+
+<liferay-frontend:management-bar
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-account-item-selector-web/src/main/resources/META-INF/resources/account_user_item_selector.jsp:27]: @andrea-ale-sbarra this is an old pattern that is deprecated please update with latest

**Review by marco-leo — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
use <clay:management-toolbar

**Comment by marco-leo** [comment]
@andrea-ale-sbarra please send this PR on 7.1.x. this won't be merge in master as there is an account refactor going on

**Pull request closed without merging**